pkg/apis/core/annotation_key_constants.go

```diff
@@ -45,11 +45,14 @@ const (
 	// to one container of a pod.
 	SeccompContainerAnnotationKeyPrefix string = "container.seccomp.security.alpha.kubernetes.io/"
```

Comment:
Should the annotation keys also be deprecated here?

Reply:
I was thinking on deprecating the keys on the same PR that implements the new functionality.
The main reason for doing this as a separate PR, is to solve the name collision between the `SeccompProfileRuntimeDefault` which represents the annotation to be deprecated, and the new `SeccompProfileRuntimeDefault` which represents the new `SeccompProfileType`.

(I will amend the PR description to include this info)